code/modules/admin/IsBanned.dm

```diff
@@ -5,7 +5,7 @@
 		log_access("Failed Login (invalid data): [key] [address]-[computer_id]")
 		return list("reason"="invalid login data", "desc"="Error: Could not check ban status, Please try again. Error message: Your computer provided invalid or blank information to the server on connection (byond username, IP, and Computer ID.) Provided information for reference: Username:'[key]' IP:'[address]' Computer ID:'[computer_id]'. (If you continue to get this error, please restart byond or contact byond support.)")
 	
-	if (computer_id == 2147783647) //this cid causes stickybans to go haywire 
+	if (computer_id == 2147483647) //this cid causes stickybans to go haywire 

```

Comment:
oh wait, this is 2^31 - 1. 
in that case, you should change this to `~(1 << 31)`.